src/ActionsImporter/Commands/AzureDevOps/Common.cs

```diff
@@ -28,9 +28,15 @@ public static class Common
         IsRequired = false,
     };
 
-    public static readonly Option<int> PipelineId = new(new[] { "--pipeline-id", "-i" })
+    public static readonly Option<int> PipelineId = new(new[] { "-i", "--pipeline-id" })
```

Comment:
Does the ordering of these options address #22 ?

Reply:
Sorry pulling this into draft, going to get all the changes in Valet first get the approvals and then mirror them over here. I was swaping them for an unrelated issue I was having with pipeline id but let me snag that bug!